website/docs/cloud-docs/api-docs/project-team-access.mdx

```diff
@@ -8,6 +8,14 @@ The team access APIs are used to associate a team to permissions on a project. A
 
 Any member of an organization can view team access relative to their own team memberships, including secret teams of which they are a member. Organization owners and project admins can modify team access or view the full set of secret team accesses. The organization token and the owners team token can act as an owner on these endpoints. ([More about permissions.](/cloud-docs/users-teams-organizations/permissions))
 
+## Project Team Access Levels
+| Access Level    | Description                                                                                                                                                      |
+|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `read`            | Read project and Read workspace access role on project workspaces                                                                                                |
+| `plan`  | Read project and Plan workspace access role on project workspaces                                                                                                |
+| `write` | Read project and Write workspace access role on project workspaces                                                                                               |
+| `admin`           | Admin project, Admin workspace access role on project workspaces, create workspaces within project, move workspaces between projects, manage project team access |
```

Comment:
I'd suggest using parenthesis to make it clear that it's not admin access + these other things, but that admin access includes these things
```suggestion
| `admin`           | Admin project, Admin workspace access role on project workspaces (create workspaces within project, move workspaces between projects, manage project team access) |
```

Reply:
Ah, I specifically used the "admin workspace access" + "specific project permissions" format to match the similar verbage for read/plan/write of "{read|plan|write} workspace access" + "specific project permissions" (there just happen to be none of the latter)

We can tweak this as we go